packaging/docker/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Dockerfile
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM ubuntu:18.04
+
+# Install dependencies
+
+RUN apt-get update && \
+	apt-get install -y wget=1.19.4-1ubuntu2 \
+	dnsutils=1:9.11.3+dfsg-1ubuntu1.2 && \
+	rm -r /var/lib/apt/lists/*
+
+# Install FoundationDB Binaries
+
+ARG FDB_VERSION
+ARG FDB_WEBSITE=https://www.foundationdb.org
+
+WORKDIR /var/fdb/tmp
+RUN wget $FDB_WEBSITE/downloads/$FDB_VERSION/linux/fdb_$FDB_VERSION.tar.gz && \
```

Comment:
You may want to consider adding a SHA256 checksum check here. Adding another argument after line 31 like:
```
ARG FDB_VERSION_SHA256SUM 
```
and then running the check in the RUN block on line 35.
```
RUN wget $FDB_WEBSITE/downloads/$FDB_VERSION/linux/fdb_$FDB_VERSION.tar.gz && \
    echo "${FDB_VERSION_SHA256SUM} fdb_${FDB_VERSION}.tar.gz" > checksum && sha256sum -c checksum && \
    tar -xzf fdb_$FDB_VERSION.tar.gz --strip-components=1 && \
```

Then running:

```
docker build . -t fdb:latest --build-arg FDB_VERSION=5.2.5 --build-arg FDB_VERSION_SHA256SUM=275b9f541590d145136061cee6427103b347e1ff0e851b6556671e46102c1ad9
```
Just a "maybe nice to have".